ostd/src/arch/loongarch/mm/mod.rs

```diff
@@ -110,10 +111,6 @@ pub(crate) unsafe fn sync_dma_range(_range: Range<Vaddr>, _direction: DmaDirecti
     unimplemented!("DMA synchronization is unimplemented in LoongArch64")
 }
 
-#[derive(Clone, Copy, Pod, Default)]
-#[repr(C)]
-pub(crate) struct PageTableEntry(usize);
-
 /// Activates the given root-level page table.
```

Comment:
Consistency: If you decide to move `PageTableEntry` below `activate_page_table`/`current_page_table_paddr`, please do so for all architectures. Otherwise, revert this change for `LoongArch`.